tests/page-objects/search/search.wdio.page.js

```diff
@@ -0,0 +1,25 @@
+const searchBox = () => $('input#freetext');
+const searchIcon = () => $('span.fa-search');
+const clearContacts = () => $('i.fa-undo');
+
+//click contact search box
+const performSearch = async (searchString) => {
+  await (await searchBox()).click();
+  await (await searchBox()).addValue(searchString);
+  await (await searchIcon()).click();
+  // After the button is pressed there can be a slight delay before the AJAX call 
+  // is made and the search spinner shows up hence we just need to wait for a bit before moving forward
+  await browser.pause(1000);
```

Comment:
We normally try to avoid random wait/pause but rather be specific on what we want to wait for. Maybe `someElement.waitForDisplayed({})`, which can also be used for wait for element to disappear with `r{everse:true}'?

Reply:
I feel using a fixed wait time here actually makes sense because this wait is not dependent on any network call, it's just to make sure that after we click, we give enough time for the search AJAX call to trigger. Maybe waiting on spinner will make it more complex. Thoughts?

Reply:
I think if it is the spinner we are waiting to disappear we could use that. `$('.loader'))`; which we probably need to do in many other areas, thus might need a sort of utility function for that. But not a biggie, so feel free to merge.